impl/client/src/main/javascript/web/dojo/pentaho/common/TemplatePicker.js

```diff
@@ -96,7 +111,27 @@ define(["dojo/_base/declare", "dijit/_WidgetBase", "dijit/_Templated", "dojo/on"
                 a11yUtil.makeAccessibleActionButton(this.templateImg2),
                 a11yUtil.makeAccessibleActionButton(this.templateImg3),
                 a11yUtil.makeAccessibleActionButton(this.templateImg4),
-                a11yUtil.makeAccessibleActionButton(this.templateImg5)
+                a11yUtil.makeAccessibleActionButton(this.templateImg5),
+
+                //Responsive implementation
```

Comment:
no change necesssary
DRY
can't you create a helper function or a loop like it does in the `showPage`

```
            showPage: function () {
              var start = this.pageNo * 6;
              for (var idx = 0; idx < 6; idx++) {

                if (idx + start < this.templates.length) {
                  this['templateImg' + idx].src = this.templates[idx + start].imagePath;
                  this['templateName' + idx].innerHTML = this.templates[idx + start].name;
                  domClass.remove(this['templateImg' + idx], 'hidden');
                  domClass.add(this['templateName' + idx], 'fade');
```

Reply:
I had assumed given that the original authors had made the loop for showPage and then manually listed each entry for the `own(...)` function, that it wasn't possible to iterate there. I know I cannot put a loop inside of that function, but let me see if I can make multiple calls to own...

Reply:
huh. I shouldn't assume. looks like I _can_ make multiple calls... I'll refactor into a loop